### PR TITLE
Feature/179434417 add more files for pride rebased core

### PR DESCRIPTION
**Pull request by lingyun1010**
We add two more files for proteomic experiment    
1. the unprocessed raw output file (`accession-proteinGroups.txt`)
2. the parameter file (`accession.mqpar.xml`)

### CODE REVIEW

File: src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentDownloadSupplier.java [hunk 1]
```diff
@@ -17,11 +18,7 @@
 import uk.ac.ebi.atlas.model.experiment.differential.microarray.MicroarrayExperiment;
 import uk.ac.ebi.atlas.profiles.ProfileStreamFilter;
 import uk.ac.ebi.atlas.profiles.baseline.BaselineProfileStreamOptions;
-import uk.ac.ebi.atlas.profiles.stream.ProteomicsBaselineProfileStreamFactory;
-import uk.ac.ebi.atlas.profiles.stream.RnaSeqBaselineProfileStreamFactory;
-import uk.ac.ebi.atlas.profiles.stream.MicroarrayProfileStreamFactory;
-import uk.ac.ebi.atlas.profiles.stream.BulkDifferentialProfileStreamFactory;
-import uk.ac.ebi.atlas.profiles.stream.ProfileStreamFactory;
+import uk.ac.ebi.atlas.profiles.stream.*;
```
Comment (upendrakumbham) [inline src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentDownloadSupplier.java:21]: Remove *  package import. Better go with specific imports.

File: src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java [hunk 2]
```diff
@@ -60,6 +63,7 @@ public StaticFilesDownload(DataFileHub dataFileHub) {
                             "Heatmap of aggregated expression data")));
         }
 
+
```
Comment (upendrakumbham) [inline src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java:66]: I think it is extra space.

Comment (upendrakumbham) [reply src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java:66]: Remove it.

File: src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java [hunk 3]
```diff
@@ -69,6 +73,29 @@ public StaticFilesDownload(DataFileHub dataFileHub) {
...
+        Path parameterFile = dataFileHub.getExperimentMageTabDirLocation()
+                .resolve(experiment.getAccession())
+                .resolve(experiment.getAccession() + ".mqpar.xml");
+        if (parameterFile.toFile().exists()) {
+            externallyAvailableContentBuilder.add(new ExternallyAvailableContent(
+                    PARAMETER_FILE_URL.replaceAll("\\{experimentAccession}", experiment.getAccession()),
+                    ExternallyAvailableContent.Description.create(
+                            "icon-clustered-heatmap",
+                            "All input parameters to run MaxQuant")));
+        }
```
Comment (upendrakumbham) [inline src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java:86]: Use `var ` in applicable places.

File: src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java [hunk 4]
```diff
@@ -91,6 +118,18 @@ public String downloadSummaryPdf(@PathVariable String experimentAccession, @Path
...
         }
+
+        @RequestMapping(value = PARAMETER_FILE_URL)
+        public String downloadParameterFile(@PathVariable String experimentAccession) {
+            String path = MessageFormat.format("/expdata/{0}/{0}.mqpar.xml", experimentAccession);
+            return "forward:" + path;
+        }
+
+        @RequestMapping(value = RAW_MAXQUANT_URL)
+        public String downloadRawMaxquant(@PathVariable String experimentAccession) {
```
Comment (upendrakumbham) [inline src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java:129]: Use` var` here as well.

File: src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java [hunk 5]
```diff
@@ -69,6 +73,29 @@ public StaticFilesDownload(DataFileHub dataFileHub) {
...
+                .resolve(experiment.getAccession() + ".mqpar.xml");
+        if (parameterFile.toFile().exists()) {
+            externallyAvailableContentBuilder.add(new ExternallyAvailableContent(
+                    PARAMETER_FILE_URL.replaceAll("\\{experimentAccession}", experiment.getAccession()),
+                    ExternallyAvailableContent.Description.create(
+                            "icon-clustered-heatmap",
+                            "All input parameters to run MaxQuant")));
+        }
+
+        Path rawMaxquant = dataFileHub.getExperimentMageTabDirLocation()
```
Comment (alfonsomunozpomer) [inline src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java:88]: `rawMaxQuant`

File: src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java [hunk 6]
```diff
@@ -24,6 +24,9 @@
     private static final String R_DATA_URL = URL_BASE + "-atlasExperimentSummary.Rdata";
     private static final String HEATMAP_URL = URL_BASE + "-heatmap.pdf";
     protected static final String SUMMARY_PDF_URL = "experiments-content/{experimentAccession}/static/{fileName}";
+    private static final String PARAMETER_FILE_URL = URL_BASE + ".mqpar.xml";
+    private static final String RAW_MAXQUANT_URL = URL_BASE + "-proteinGroups.txt";
```
Comment (alfonsomunozpomer) [inline src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java:28]: `RAW_MAX_QUANT_URL`

File: src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java (hunk 4 above)
Comment (alfonsomunozpomer) [reply src/main/java/uk/ac/ebi/atlas/experimentpage/StaticFilesDownload.java:129]: `downloadRawMaxQuant`

File: src/main/java/uk/ac/ebi/atlas/resource/DataFileHub.java [hunk 7]
```diff
@@ -44,6 +44,9 @@
     protected static final String IDF_FILE_PATH_TEMPLATE = "{0}/{0}.idf.txt";
     protected static final String SUMMARY_PDF_FILE_PATH_TEMPLATE = "{0}/{0}{1}Summary_ExpressionAtlas{2}pdf";
 
+    protected static final String PROTEOMICS_PARAMETER_FILE_PATH_TEMPLATE = "{0}/{0}.mqpar.xml";
+    protected static final String PROTEOMICS_RAW_MAXQAUNT_PATH_TEMPLATE = "{0}/{0}-proteinGroups.txt";
```
Comment (alfonsomunozpomer) [inline src/main/java/uk/ac/ebi/atlas/resource/DataFileHub.java:48]: `PROTEOMICS_RAW_MAX_QUANT_PATH_TEMPLATE`

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
Some typos left to fix.

**Review by upendrakumbham — APPROVED** [review]
All Good! Please merge.

**Pull request merged**